src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -259,6 +259,20 @@ Apply to thead to make headers not bold.
     padding-top: 0;
 }
 
+.question-recommended-length {
+    color: gray;
+}
+
+.question-recommended-length > span {
+    margin-left: 7px;
+    border-bottom: 1px dotted;
```

Comment:
Yup, I think it's better to use that as its semantics better fit what we are trying to achieve